news/models.py

```diff
@@ -214,3 +214,13 @@ class Meta:
         permissions = (
             ("cancel_ticket", "Can cancel and reactivate all event tickets"),
         )
+
+    @property
+    def name(self):
+        # For backwards compatibility, name is no longer set
```

Comment:
This comment should probably be above the `_name` field instead of here, with a docstring for what the function does, here instead. Same goes for the `email` property below with the `_email` field.